### DISCUSSION TITLE
Remix Austin Meetup website mock ups

### DISCUSSION THREAD

**Discussion by sethdavis512** [remix-austin/new-remixaustin-com#18] [General]
Let's chit chat about the future design of Remix Austin Meetup's website here.

Link to mockups: https://www.figma.com/design/c3TgqL3EPtx4lkzb7Ey5z4/Remix-Austin-Meetup-Website?node-id=0-1&t=Q6uuBZYHeH10ouR4-0

**Comment by sethdavis512** [comment]
What are some things from the current site ([remixaustin.com](www.remixaustin.com)) that you definitely think we should keep for the new site?

**Comment by sethdavis512** [comment]
Here's our current set of routes:

<img width="250" alt="image" src="https://github.com/user-attachments/assets/cdbc133e-0362-444f-8acb-278b6a262725">

DISCLAIMER: Don't want to implement immediately...but...what other routes/ideas should we include?

**Reply by bradgarropy** [reply]
> DISCLAIMER: Don't want to implement immediately...

![talking-to-me-cody-garbrandt-gif-by-ufc](https://github.com/user-attachments/assets/4964cbe1-048b-4bb7-9de6-c813397b8d87)

Wish I could attend the next Remix Rodeo meetup so we could talk about this in person, but I've got a conflict.
